src/js/view/controls/tizen/tizen-controls.ts

```diff
@@ -185,19 +155,172 @@ class TizenControls extends Controls {
     }
 
     disable(model: ViewModel): void {
+        this.model = null;
+
+        if (this.apiEnabled) {
+            this.api.off(null, null, this);
+            this.api = null;
+        }
+
         if (this.keydownCallback) {
             document.removeEventListener('keydown', this.keydownCallback);
         }
```

Comment:
You can call `document.removeEventListener` using the callback or method whether or not it's already added as a listener.
```suggestion
        document.removeEventListener('keydown', this.handleKeydown);
```